Icon/Icons/WarningIcon.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import Icon from '../../Icon';
 
-const WarningIcon = ({ color, size }) =>
+const WarningIcon = ({ color, size }) => (
```

Comment:
Ah, great one. Yeah, ESLint was raising this block as a warning. Perhaps we could adjust ESLint a little here for this one? 🤔